app/helpers/race_reports_helper.rb

```diff
@@ -0,0 +1,7 @@
+module RaceReportsHelper
+  def available_race_reports
```

Comment:
This method will look at the `/app/views/pages/race_reports` directory and dynamically return an array of available race report years, such as:
```
["2020", "2019"]
```